README.md

```diff
@@ -1,104 +1,215 @@
+  -o             path to store output file [required]
 ```
 
-## `rarity`
-
-Calculate TF-IDF information from sequencing data. Two counting Bloom filters will be stored. One for *k*-mer/seed counts (TF) and other for depths (DF). The number of reads ($N$) will also be printed to stdout. To compute the TF-IDF of an element with $h$ as its hash array, use this formula:
+- Refer to the `count` module's notes for information about the parameters.
 
-$$ \text{TF-IDF}(h) = TF(h) * log \dfrac{N + 1}{DF(h) + 1} $$
-
-Note that the $+1$ in the $log$ operation is to prevent division by zero.
+## `query`
 
-### Usage
+Converts information from a BF/CBF to TSV format.
 
 ```
-rarity [--help] [--version] [-k VAR] [-s VAR] -f VAR [-e VAR] [-b VAR] [--long] [-t VAR] -o VAR reads
+usage: query [-h] -b B [-s S] -o O data [data ...]
 
-Positional arguments:
-  reads          path to sequencing data file(s) [nargs: 0 or more] [required]
+positional arguments:
+  data        path to query data
 
-Optional arguments:
-  -h, --help     shows help message and exits
-  -v, --version  prints version information and exits
-  -k             k-mer length
-  -s             path to spaced seeds file (one per line, if -k not specified)
-  -f             path to k-mer spectrum file (from ntCard) [required]
-  -e             target output false positive rate [nargs=0..1] [default: 0.0001]
-  -b             output CBF size (bytes)
-  --long         optimize for long read data
-  -t             number of threads [nargs=0..1] [default: 1]
-  -o             prefix for storing output counting bloom filters [required]
+options:
+  -h, --help  show this help message and exit
+  -b B        path to BF/CBF file
+  -s S        path to spaced seeds file
+  -o O        path to output TSV file
 ```
 
+- The input data can be any FASTA, FASTQ, or BAM. Compressed files are supported. To read from stdin, use `-`.
+- If the data structure contains spaced seed information, pass the path to the spaced seeds text file to `-s`.
+
 ## `hist`
 
-### Usage
+Fits a mixture model to the *k*-mer count histogram to estimate *k*-mer error and heterozygous probabilities, genome size, and thresholds useful for the `filter` command.
 
 ```
-hist [--help] [--version] histogram
-
-Positional arguments:
-  histogram      path to k-mer spectrum file (from ntCard) [required]
+usage: main.py [-h]
+               [-f {asciidoc,double_grid,double_outline,fancy_grid,fancy_outline,github,grid,heavy_grid,heavy_outline,html,jira,latex,latex_booktabs,latex_longtable,latex_raw,mediawiki,mixed_grid,mixed_outline,moinmoin,orgtbl,outline,pipe,plain,presto,pretty,psql,rounded_grid,rounded_outline,rst,simple,simple_grid,simple_outline,textile,tsv,unsafehtml,youtrack}]
+               [-m STYLE] [-t TITLE] [--y-log | --no-y-log] [-r PLOT_RANGE] [-o PLOT] [--probs PROBS] [--fit-gif FIT_GIF] [-c CONFIG] [--no-model]
+               path
+
+positional arguments:
+  path                  k-mer spectrum file
```

Comment:
Think would be important to say "from ntCard" still here

Reply:
Fixed in 2057c6844555100d86789b492ba3d2a316aa628a